001-OFEP-cloud-native-pattern.md

```diff
@@ -120,11 +131,12 @@ In order to enable host containers to consume the sidecar then there should be m
 There was an initial proposal to incorporate the AF_LOCAL/AF_UNIX socket family and within that family, we should decide whether is a need to support SOCK_STREAM and SOCK_DGRAM, I would initially suggest only supporting SOCK_STREAM.
 This would enable us to further layer HTTP protocol support on top where required.
 
-<img src="images/001-05.png" width="300">
+![unlabelled_image](images/001-05.png "unlabelled_image")
+
 
 ## Flow
 The below illustration has been updated also to reflect the current thinking around the initialisation flow of the flagging system.
 
 That said there are some learnings from Istio and concerns around side car overhead - namely around upgrading and maintenance. As such it is worth exploring a pattern for rolling or upgrading sidecars, as the implication is that this will force a deployment rollout due to the change on the deployment object ( and other resource types sts/ds)
 
-<img src="images/001-06.png" width="300">
\ No newline at end of file
+![unlabelled_image](images/001-06.png "unlabelled_image")
```

Comment:
If landing this is important w/o providing alt text, at least file a ticket for adding alt text for these things.